### PR TITLE
add persian doc

### PR DESCRIPTION
**Pull request by hossein-raeisi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<p align="center">
+  <img src="./pictures/bazaar-pay-logo-english.png" alt="Vizdom"/>
+</p>
+
+Payment service usage technical documentation
+
+BazaarPay provides multiple payment solutions For merchants. and SDKs for easy and faster adaptation, including:
```
Comment (massood) [inline README.md]: BazaarPay provides merchants with multiple payment solutions, and SDKs for easy and fast adoption, including:

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+- [Android SDK](https://github.com/cafebazaar/BazaarPay)
+- [Web SDK](https://www.npmjs.com/package/@cafebazaar/payment-sdk)
+- SDK less payment
+
+We also provide [direct payment service](./persian/DirectPay.md) for easier payments and better user experience.
+
+Merchants can also [integrate BazaarPay's wallet](./persian/Wallet.md) in their applications.
```
Comment (massood) [inline README.md]: in -> into

BazaarPay's wallet -> BazaarPay wallet

File: README.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<p align="center">
+  <img src="./pictures/bazaar-pay-logo-english.png" alt="Vizdom"/>
+</p>
+
+Payment service usage technical documentation
```
Comment (massood) [inline README.md]: BazaarPay service technical documentation

/?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+BazaarPay provides multiple payment solutions For merchants. and SDKs for easy and faster adaptation, including:
+
+- [Android SDK](https://github.com/cafebazaar/BazaarPay)
+- [Web SDK](https://www.npmjs.com/package/@cafebazaar/payment-sdk)
+- SDK less payment
```
Comment (massood) [inline README.md]: SDK-less payment

File: persian/Payment.md [hunk 5]
```diff
@@ -0,0 +1,383 @@
...
+{"status":"paid_committed"}
+```
+
+#### Checkouts Status Report:
+
+```yaml
+paths:
+  { base_url }/get-checkouts-status/
+    post:
+      summary: commit
```
Comment (hossein-raeisi) [inline persian/Payment.md]:
```suggestion
      summary: checkouts-status
```

File: persian/Payment.md [hunk 6]
```diff
@@ -0,0 +1,383 @@
...
+}'
+```
+
+#### Trace:
+
+```yaml
+paths:
+  { base_url }/trace/
+    post:
+      summary: commit
```
Comment (hossein-raeisi) [inline persian/Payment.md]:
```suggestion
      summary: trace
```

**Pull request merged**